Sources/NIOCore/ByteBuffer-core.swift

```diff
@@ -150,6 +150,35 @@ public struct ByteBufferAllocator: Sendable {
 ///  - `Foundation`'s `Data`
 ///  - `[UInt8]` and generally any `Collection` of `UInt8`
 ///
+/// **The best and easiest way to read and convert `ByteBuffer` into a `String`,  `Array<UInt8>`, or `Data` is by
```

Comment:
We probably don't need the extra bolding, you can remove the `**`. Also we can put `ByteBuffer` into double-backticks.